samples/ftd_configuration/identity_policy.yml

```diff
@@ -0,0 +1,53 @@
+- hosts: all
+  connection: httpapi
+  tasks:
+    - name: Setup an identity policy
+      ftd_configuration:
+        operation: upsertIdentityPolicy
+        data:
+          name: NGFW-Default-Identity-Policy
+          defaultIdentityRule:
+            authAction: PASSIVE
+            type: defaultidentityrule
+          activeAuthPort: "885"
```

Comment:
do we need quotes here?

Reply:
Yes. Otherwise, Ansible thinks that it's an integer and raises a validation error.